### PR TITLE
Internationalization of the App to Arabic

### PR DESCRIPTION
**Pull request by rjmangubat23**
## Issue

- Available is only English version 

## Changes

- Add Arabic version to app
- Add Fixes for Names

### CODE REVIEW

File: app/src/main/java/org/idpass/smartscanner/settings/SettingsActivity.kt [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import java.util.*
+
+
+class SettingsActivity : AppCompatActivity() {
+    @SuppressLint("CommitPrefEdits")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_settings_languages)
+
+        val engilshpic = findViewById<ImageView>(R.id.engilshpic)
```
Comment (arnellebalane) [inline app/src/main/java/org/idpass/smartscanner/settings/SettingsActivity.kt:41]: @rjmangubat23 Typo for `english` here 😅

File: app/src/main/res/layout/activity_settings_languages.xml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        android:orientation="vertical"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.049"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView"
+        app:layout_constraintVertical_bias="0.033">
+
+        <LinearLayout
+            android:id="@+id/engilshlayout"
```
Comment (arnellebalane) [inline app/src/main/res/layout/activity_settings_languages.xml:66]: @rjmangubat23 Typo for `english` here 😅

File: app/src/main/res/layout/activity_settings_languages.xml [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                android:layout_gravity="center"
+                android:src="@drawable/ic_settings_menu" />
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:layout_marginStart="8dp"
+                android:text="@string/english" />
+            <ImageView
+                android:id="@+id/engilshpic"
```
Comment (arnellebalane) [inline app/src/main/res/layout/activity_settings_languages.xml:84]: @rjmangubat23 Typo for `english` here 😅

File: core-lib/src/main/java/org/idpass/smartscanner/lib/nfc/NFCResult.kt [hunk 4]
```diff
@@ -42,4 +48,50 @@ data class NFCResult(
...
+        fun formatResult(passport: Passport?, mrzInfo: MRZInfo? = null): NFCResult {
+            val personDetails = passport?.personDetails
+            val additionalPersonDetails = passport?.additionalPersonDetails
+            val additionalDocumentDetails = passport?.additionalDocumentDetails
+            // Note: Get proper names
+            // NFCResult.nameOfHolder --> LAST_NAME<<GIVEN_NAMES (RTL for Arabic)
+            // we split nameOfHolder to two parts separated by '<<' (double chevron)
+            val nameParts  = additionalPersonDetails?.nameOfHolder?.split("<<")?.toMutableList()
+            // then first part of nameOfHolder should contain LAST_NAME
+            val surname = nameParts?.get(0)
```
Comment (arnellebalane) [inline core-lib/src/main/java/org/idpass/smartscanner/lib/nfc/NFCResult.kt:63]: @rjmangubat23 The surname can also contain multiple words separated by `<`.

**Review by arnellebalane — CHANGES_REQUESTED** [review]

File: core-lib/src/main/java/org/idpass/smartscanner/lib/nfc/NFCResult.kt (hunk 4 above)
Comment (nabaanahith) [reply core-lib/src/main/java/org/idpass/smartscanner/lib/nfc/NFCResult.kt]: yes, the parts may has size more or less than 3 .. so we have to make it dynamically not static

**Review by arnellebalane — APPROVED** [review]

**Pull request closed without merging**